src/web/components/FluentUIProvider.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { FluentProvider, webLightTheme } from '@fluentui/react-components';
+
+interface Props {
+  children: React.ReactNode;
+}
+
+function FluentUIProvider({ children }: Props) {
```

Comment:
Lets use arrow functions for functional components

Reply:
Also, Add JSDOC

Reply:
Noted